src/main/java/fizzbuzz/Translator.java

```diff
@@ -0,0 +1,22 @@
+package fizzbuzz;
+
+import java.util.List;
+
+public class Translator implements Checker, ITranslator {
+    private List<Checker> checkers;
+    private String name;
+
+    public Translator(String name, List<Checker> checkers){
+        this.name = name;
+        this.checkers = checkers;
+    }
+
+    public boolean shouldTranslate(int number){
+        return checkers.stream().map(checker -> checker.shouldTranslate(number)).reduce(true, Boolean::logicalAnd);
```

Comment:
Does this also work for OR operator on multiple checkers?
As far as we understand, this resolves to true, if and only if all checkers return true...